apps/system/test/marionette/lib/utility_tray.js

```diff
@@ -20,6 +20,7 @@
       element: '#utility-tray',
       screen: '#screen.utility-tray',
       notifContainer: '#desktop-notifications-container',
+      firstNotification: '#desktop-notifications-container .notification',
```

Comment:
nit: You decide - I think it's better to just call this 'notification', as you could do say this.client.findElements(Selectors.firstNotification).pop() to get the last element, and that reads weirdly. On the other hand, we have form for calling this 'firstSomething' in other files. I don't like it, but it's up to you :)